### PR TITLE
DM-32897: Remove START and STOP commands.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: tests/test_command_handler.py [hunk 1]
```diff
@@ -96,6 +96,26 @@ async def test_configure(self) -> None:
...
         assert self.configuration == self.command_handler.configuration
+        # Send the same configuration again which should lead to an error.
+        await self.command_handler.handle_command(
+            command=common.Command.CONFIGURE, configuration=self.configuration
+        )
+        self.assert_response(common.ResponseCode.ALREADY_STARTED)
+        # Make sure to stop so a new config command can be sent.
+        await self.command_handler.stop_sending_telemetry()
+        # Give time to the telemetry_task to get cancelled.
+        await asyncio.sleep(0.5)
```
Comment (r-owen) [inline tests/test_command_handler.py:107]: Is it possible to make `stop_sending_telemetry()` wait until it has truly stopped before returning? That would eliminate the need for this sleep.

If not, is there some task you can await in the command handler or in the mock server, to know that the connection has been dropped? That would be more robust than waiting for a fixed amount of time.

If there is no existing natural task for this, please consider adding such a Task, Future, or asyncio.Event. If the use is only for unit tests, that's OK, just document it as such. I sometimes add a task for this that a unit test can set to a new Future and that the library code sets done if not already done when the thing happens.

**Review by r-owen — COMMENTED** [review]
I suspect you may have forgotten to push some changes. I don't see any changes to the server. I also still see tests that call stop_sending_telemetry and start_sending_telemetry; should those exist? I see you removed them from the abstract abstract command handler. Please let me know what's going on and I'll finish the review.

**Review by r-owen — APPROVED** [review]
Looks good. A nice change. You have a few tests that have "start" in the name. I gently suggest considering whether to update the names and/or brief descriptions. It's not crazy to expect the user to know that configure=start, but still...if you're that keen on that connection then it might make more sense to call the remaining command "start" instead of "configure".

File: tests/test_command_handler.py (hunk 1 above)
Comment (wvreeven) [reply tests/test_command_handler.py:107]: It turns out that the calls to `sleep` weren't necessary so I removed them all.

**Pull request merged**